Makefile

```diff
@@ -1,6 +1,22 @@
 CACHITO_COMPOSE_ENGINE ?= docker-compose
-ifeq ($(CACHITO_COMPOSE_ENGINE), docker-compose)
-	DOWN_OPTS=-v
+
+PODMAN_COMPOSE_AUTO_URL ?= https://raw.githubusercontent.com/containers/podman-compose/devel/podman_compose.py
+PODMAN_COMPOSE_TMP ?= tmp/podman_compose.py
+
+ifeq (podman-compose-auto,$(CACHITO_COMPOSE_ENGINE))
+ifeq (,$(wildcard $(PODMAN_COMPOSE_TMP)))
```

Comment:
I'd like to indent this line and the ones below, but Makefile complains of invalid syntax. I'm not sure why it's ok to indent on line 19. (Yes, I used tabs instead of leading spaces)

Reply:
How are you trying to indent these lines? Tabs or spaces?

Reply:
Tried both spaces and tabs. Both fail with the same error.

Reply:
It could be because line 19 is not getting executed. I bet if you get into that code block then it will create an error!

Reply:
I think you're right, or it's probably some bug in the Makefile parser because the line does contain the `:` character. In any case, I've "flattened" that else block for consistency.